setup.py

```diff
@@ -2,3 +2,4 @@
 
 if __name__ == "__main__":
     setup()
+    print ('I have made some changes..')
```

Comment:
I understand that you're trying the platform, so I'll reject the PR, right?
Otherwise every time we run the setup file, it will print your message :)